decidim-core/app/helpers/decidim/feature_path_helper.rb

```diff
@@ -21,6 +21,15 @@ def manage_feature_path(feature)
       feature_root_path_for(feature.manifest.admin_engine, feature)
     end
 
+    # Returns the config path for a given feature.
+    #
+    # feature - the Feature we want to find the root path for.
+    #
+    # Returns a url.
+    def edit_feature_path(feature)
+      decidim_admin.edit_feature_path(id: feature.id, participatory_process_id: feature.participatory_process.id)
+    end
+
```

Comment:
Since this helper is only used for tests, I wonder if this is the best place for it? Maybe moving it to https://github.com/decidim/decidim/blob/628fb8bff653b426b80d73b74e0010e958f26433/decidim-dev/lib/decidim/dev/test/rspec_support/feature_context.rb, or somewhere similar? Or maybe it's no big deal ¯\_(⊙︿⊙)_/¯.